cmd/meroxa/root/apps/apps.go

```diff
@@ -59,3 +66,28 @@ func (*Apps) SubCommands() []*cobra.Command {
 		builder.BuildCobraCommand(&Upgrade{}),
 	}
 }
+
+type addHeader interface {
+	AddHeader(key, value string)
+}
+
+func addTurbineHeaders(c addHeader, lang, version string) {
+	c.AddHeader("Meroxa-CLI-App-Lang", lang)
+	if lang == turbine.JavaScript {
+		version = fmt.Sprintf("%s:cli%s", version, turbine.TurbineJSVersion)
```

Comment:
What is these two values?

Reply:
the first is the turbine-js-framework version and the second is the turbine-js-cli version. they can drift.